CONTRIBUTING.md

```diff
@@ -10,14 +10,36 @@ To discuss anything please [contact Cornelius](mailto:cschum@suse.de).
 
 ## Contribute code
 
-To contribute code please open pull requests.
+To contribute code fork the repository and open pull requests with your changes.
 
 Ensure that rspec and rubocop pass locally before sending your PR and always that you add new changes.
 
 If your changes include important new features or bug fixes please add them to the [Master (unreleased) section fo the CHANGELOG.md](https://github.com/openSUSE/trollolo/blob/master/CHANGELOG.md#master-unreleased)
 
+### Set up
+
+Clone git repo: 
+
+`git clone git@github.com:openSUSE/trollolo.git`
+
+Build the gem: 
+
+`gem build trollolo.gemspec`
+
+Install the gem: 
+
+`gem install trollolo-0.2.0.gem`
+
+To test and debug changes in Interactive Ruby Shell: 
+
+`irb -Ilib -rtrollolo`
+
 ### To run rspec test
 
+First build matpotlib docker image(necessary for testing burndown charts):
```

Comment:
why do you need to do this? 😕

Reply:
![image](https://user-images.githubusercontent.com/22021150/35733425-5cb2893c-0843-11e8-923b-e87a88f38086.png)
On the first time that i ran `bundle exec rspec` it threw me the above error generated from /spec/integration/create_burndown_spec.rb file, it stated that there was no 'trollolo-matplotlib' docker image present, hence i thought adding this was necessary to build the required image.

Reply:
but this is not needed for all the test, right?

Reply:
it's not required for all the tests, just the burndown charts, as I have also mentioned.

Reply:
I just realised that in Travis we are only running the unit tests. I think this should be fixed in https://github.com/openSUSE/trollolo/issues/119

I would just change in the documentation to run `bundle exec rspec spec/unit` then. As anyway the other tests are not run in Travis. 😉

Reply:
okay i'll make the changes. :smile: